helix-rest/src/main/java/org/apache/helix/rest/server/service/VirtualTopologyGroupService.java

```diff
@@ -60,15 +62,15 @@ public class VirtualTopologyGroupService {
   private final ClusterService _clusterService;
   private final ConfigAccessor _configAccessor;
   private final HelixDataAccessor _dataAccessor;
-  private final VirtualGroupAssignmentAlgorithm _assignmentAlgorithm;
+  private VirtualGroupAssignmentAlgorithm _assignmentAlgorithm;
 
   public VirtualTopologyGroupService(HelixAdmin helixAdmin, ClusterService clusterService,
       ConfigAccessor configAccessor, HelixDataAccessor dataAccessor) {
     _helixAdmin = helixAdmin;
     _clusterService = clusterService;
     _configAccessor = configAccessor;
     _dataAccessor = dataAccessor;
-    _assignmentAlgorithm = FifoVirtualGroupAssignmentAlgorithm.getInstance();
+    _assignmentAlgorithm = FaultZoneBasedVirtualGroupAssignmentAlgorithm.getInstance(); // default assignment algorithm
```

Comment:
We should not change the algorithm default or this will break backwards compatibility.